docs/install.md

```diff
@@ -5,33 +5,33 @@ post installation to ensure the operator can function correctly.
 
 ## STS Clusters
 
-In an STS Cluster, *CredentialsRequests* are not automatically provisioned by
-the `cloud-credential-operator` and requires manual intervention by the
-cluster-admin. The credentials secret needs to be provisioned using the `ccoctl` binary.
+### Post operator installation
 
-The `aws-load-balancer-operator` also relies on the `cloud-credential-operator`
-to provision the secret for *CredentialsRequest*. And so in an STS Cluster the
-secret needs to be provisioned manually. The operator will wait until the required
-secrets are created and available.
+In an STS Cluster, `CredentialsRequest`s are not automatically provisioned by
+the **cloud-credential-operator** and the manual intervention done by the
+cluster-admin is required. IAM role and policies as well as the credentials secret need to be provisioned manually for the further consumption by the PODs.
+`ccoctl` binary can be used to facilitate this task.
```

Comment:
Your latest changes are an improvement.  Thanks!  

One thing I had forgotten is that OLM doesn't support CredentialsRequest CRs, so creating the secret or CredentialsRequest for the operator is _always_ required, and creating the secret or CredentialsRequest for the subordinate controller is only _sometimes_ required (namely for STS clusters).  So during my earlier review, I had been thinking that the normal, non-STS installation flow automated all the credentials management (as is the case for CVO-managed operators), and that misunderstanding caused some of my confusion.  

The structure of the documentation still seems a little strange to me—it seems to me that the content of `prerequisites.md` is really the installation instructions, and the content of `install.md` is really the post-installation instructions for STS clusters—but it's at least clearer now.

Reply:
> The structure of the documentation still seems a little strange to me—it seems to me that the content of prerequisites.md is really the installation instructions, and the content of install.md is really the post-installation instructions for STS clusters—but it's at least clearer now.

Yes, I found the naming of the files weird too. Just didn't want to change the docs completely, wanted to make it more incremental. However I agree, we need to find better names to avoid the confusion.